### ISSUE TITLE
Loading RPG_RT savegames triggers screen shake

### ISSUE THREAD

**Issue by fdelapena** [EasyRPG/Player#1668]
> Hi, I'm having some troubles after the last update. Now some of the games I was playing, like Yume Nikki, don't stop shaking after I load the save file.

[Yume Nikki easyrpg_log.txt](https://github.com/EasyRPG/Player/files/2931978/0_easyrpg_log.txt)
[Yume Nikki savegames](https://github.com/EasyRPG/Player/files/2931979/savegames.zip)

[OFF translation 2.0 easyrpg_log.txt](https://github.com/EasyRPG/Player/files/2942863/easyrpg_log.txt)
[OFF translation 2.0 savegames](https://github.com/EasyRPG/Player/files/2942864/off_translation_2.0_savegames.zip)

**Comment by Ghabry** [comment]
The chunks say:
```
    <shake_continuous>T</shake_continuous>
    <shake_strength>1</shake_strength>
    <shake_speed>9</shake_speed>
```

**Comment by Ghabry** [comment]
hard to say where the problem is because my old savegames don't shake in YN

**Comment by carstene1ns** [comment]
I just built Player@https://github.com/EasyRPG/Player/commit/e410c9c55f6b841017e43f60f0fa567ae9841ec2 with https://github.com/EasyRPG/liblcf/commit/5b52d85846016855e87435f649b96f88617e6566 to recreate the issue and it does not happen. Also, when re-saving the shake is gone...

**Comment by fmatthew5876** [comment]
If it has the shake_continuous chunk as 1 then the game will shake...

Not sure how it got that chunk but if it's there then it's doing what it is supposed to do.

**Comment by fdelapena** [comment]
Savegames are created in PC with RPG_RT.exe, confirmed by the reporter.

Added OFF translation 2.0 savegames and log.

**Comment by fmatthew5876** [comment]
Are we absolutely sure the reporter created the save game with RPG_RT and not player? 

If the save game was really created by Player, then I'd bet this was the same as #1744. The save game has the continuous chunk set to true, so I don't know what else to say..

**Comment by Ghabry** [comment]
Well assuming the EasyRPG version used to play on the PC is old enough (0.5.4, not a nightly) then we could proof if it is a EasyRPG or RPG_RT save by simply saving-and-loading it through some lcf2lcf tool because of the way how chunks were written in 0.5.4.

**Comment by carstene1ns** [comment]
But this might be affected by liblcf#305.

**Comment by Ghabry** [comment]
Considering that we never got another bug report about this even though this made popular games unplayable I consider this bogus and close it without further investigation.

Maybe fixed by #1743

**Comment by Nootboot64** [comment]
I get that too! Sorry for taking so long to find this, but on my PSVITA, I get screen shake after loading my save file, specifically when I had transferred it from my PC. I get this on OFF, but it is the only game I've tested.

**Comment by carstene1ns** [comment]
Please send us the savegame (saved by RPG_RT).

**Comment by Nootboot64** [comment]
Here, I think this is what you meant:
[Save01.zip](https://github.com/EasyRPG/Player/files/3752933/Save01.zip)

**Comment by Ghabry** [comment]
Shame, so we actually didn't fix it? :/

A second savegame is available here: https://community.easyrpg.org/t/saves-imported-that-are-not-initially-made-in-easyrpg-causing-screen-shaking/755

**Comment by Ghabry** [comment]
Okay this is interesting.
When I load the attached Save01.zip by Nootboot in OFF it shakes in Player but not in RPG_RT. shake_continious is T. Maybe there is a second variable required in the savefile to really start a shake?

**Comment by Ghabry** [comment]
Found it already. >.>
When you call "Start shake" it sets shake\_continuous to T and shake\_time\_left to 65535. The time is decrementing each frame and when it reaches 0 (confirmed with save editing by changing it to 100) the shake stops when it is at 0 (!)

**Comment by Nootboot64** [comment]
So until there's a permanent fix, couldn't I just get the save with the edits and use that?

**Comment by fmatthew5876** [comment]
Fixed in #1922
